tests/test_jobs.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from pybuildkite.buildkite import BuildKite, NoAcccessTokenException
+from pybuildkite.jobs import Jobs
+from unittest.mock import Mock
+
+
+class TestBuildkite:
```

Comment:
TestJobs class is currently being built here. https://github.com/pyasi/pybuildkite/pull/7  Maybe you two could work together? I like what you've done with the mocking and fixture here

Reply:
I'll wait till theirs is in, then rebase :)

Reply:
Typically I wouldn't bother with the outer class either, but that seemed to be the style of the others, so I mimicked it